extensions/extensions-docker/aissemble-spark/src/main/resources/docker/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Script for creating base Spark Docker image
 #
 ARG SPARK_VERSION
-FROM apache/spark-py:v${SPARK_VERSION}
+FROM apache/spark:${SPARK_VERSION}-scala2.12-java17-python3-ubuntu
```

Comment:
A: This may not be worth it, but we could pull the scala version from `build-parent` like we're doing with Spark.  The only complexity is that we need just the major/minor numbers and not the full `2.12.13`.  A cheap option could be just to have a sister property to `version.scala` that lives _right_ next to it in `build-parent` and is set to `2.12`.  At that point it's pretty likely that someone bumping up the scala version to a new `2.x` version would see and also update that value.

Reply:
I like it. Ill go with suggested